### PR TITLE
add get redirect without html

### PR DESCRIPTION
**Pull request by amaliwanag2**

### CODE REVIEW

**Comment by amaliwanag2** [comment]
end 2 end tests passed on browser stack, need to adjust timeout in tab-e2e

File: src/utils/__tests__/redirect.test.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  })
+
+  it('redirects to the Chrome Web Store when the browser is Chrome on mobile', () => {
+    expect.assertions(1)
+    getBrowserInfo.mockReturnValue(createMockBrowserInfo('chrome', true))
+    const myRedirect = directToAppExtension()
+    expect(myRedirect).toEqual(chromeExtensionURL)
+  })
+
+  it('redirects to the Firefox Add-ons page when the browser is Firefox on desktop', () => {
```
Comment (kmjennison) [inline src/utils/__tests__/redirect.test.js:46]: Update test description

File: src/utils/__tests__/redirect.test.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    expect.assertions(1)
+    getBrowserInfo.mockReturnValue(createMockBrowserInfo('firefox', false))
+    const myRedirect = directToAppExtension()
+    expect(myRedirect).toEqual('/')
+  })
+
+  it('redirects to the Firefox Add-ons page when the browser is Firefox on mobile', () => {
```
Comment (kmjennison) [inline src/utils/__tests__/redirect.test.js:53]: Update test description

File: src/pages/__tests__/get.test.js [hunk 3]
```diff
@@ -1,138 +1,40 @@
...
-    getBrowserInfo.mockImplementation(() => {
-      throw new Error('I could not browse browser :(')
-    })
+  it('redirects to the proper browser extension', () => {
     const GetExtensionRedirectPage = require('../get').default
     const mockProps = getMockProps()
     shallow(<GetExtensionRedirectPage {...mockProps} />)
-    expect(redirect).toHaveBeenCalledWith(homeURL)
+    expect(redirect).toHaveBeenCalled()
+    expect(directToAppExtension).toHaveBeenCalled()
```
Comment (kmjennison) [inline src/pages/__tests__/get.test.js:33]: A better test: mock a return URL for `directToAppExtension`, then assert redirect is called with that URL.

File: src/pages/__tests__/cats-get.test.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    const mockProps = getMockProps()
+    shallow(<GetExtensionRedirectPage {...mockProps} />)
+  })
+
+  it('redirects to the proper browser extension', () => {
+    const GetExtensionRedirectPage = require('../cats/get').default
+    const mockProps = getMockProps()
+    shallow(<GetExtensionRedirectPage {...mockProps} />)
+    expect(redirect).toHaveBeenCalled()
+    expect(directToAppExtension).toHaveBeenCalled()
```
Comment (kmjennison) [inline src/pages/__tests__/cats-get.test.js:33]: A better test: mock a return URL for `directToAppExtension`, then assert redirect is called with that URL.

**Review by kmjennison — CHANGES_REQUESTED** [review]

**Comment by amaliwanag2** [comment]
@kmjennison you can see the ui on dev at http://dev-tab2017.gladly.io/cats/getHTML

**Review by kmjennison — CHANGES_REQUESTED** [review]
Functional problem: the unsupported browser redirects to the main homepage but should redirect to `/cats/`.

Aesthetic preference for users: instead of `/cats/getHTML/`, how about: `/cats/get/intro/`?

**Comment by kmjennison** [comment]
Content for the timer redirect looks great, BTW 👍

File: src/pages/cats/get.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+          : null
+      if (referrerId) {
+        localStorageMgr.setItem(
+          STORAGE_REFERRAL_DATA_REFERRING_CHANNEL,
+          referrerId
+        )
+      }
+
+      // Based on the browser, go to the appropriate extension page.
+      redirect(directToAppExtension())
```
Comment (kmjennison) [inline src/pages/cats/get.js:30]: Needs to go to cats page: `directToAppExtension(true)`

**Review by kmjennison — CHANGES_REQUESTED** [review]

**Review by kmjennison — APPROVED** [review]
LGTM 🦊

**Pull request merged**